python/tvm/topi/nn/conv3d.py

```diff
@@ -178,7 +207,12 @@ def conv3d_ndhwc(Input, Filter, stride, padding, dilation, out_dtype="float32"):
         ),
         name="Conv3dOutput",
         tag="conv3d_ndhwc",
+        attrs={"layout_free_placeholders": [Filter]},
     )
+
+    if auto_scheduler_rewritten_layout:
+        Output = auto_scheduler.rewrite_compute_body(Output, auto_scheduler_rewritten_layout)
+
     return Output
```

Comment:
How about we move the `auto_scheduler_rewritten_layout` to be checked inside the `rewrite_compute_body`?
If it is empty, return the original compute.
```suggestion
    return auto_scheduler.rewrite_compute_body(Output, auto_scheduler_rewritten_layout)
```

Reply:
The current style is clearer. For a person who do not want to learn auto-scheduler, he/she can simply skip any code under branch `if auto_scheduler_rewritten_layout`.
In your style, he/she has to check the code of `auto_scheduler.rewrite_compute_body` to know what's happening.